app/konnect/org-management/okta-idp.md

```diff
@@ -271,9 +266,6 @@ in with your Okta credentials.
     If a group-to-team mapping exists, the user is automatically provisioned with
     a {{site.konnect_saas}} account with the relevant team membership.
 
-1. Log out of this account, and log back in with a {{site.konnect_short_name}}
```

Comment:
@smritikjaggi I also removed this line since I think this one is implying that you can still log in with a Konnect admin org owner. Is this accurate?

Reply:
I think it looks good! Thank you!